### PR TITLE
Adding Function to Calculate Various SIRS Irradiance Measurements

### PR DESCRIPTION
**Pull request by ajsockol**

### CODE REVIEW

File: act/retrievals/sirs_calculations.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
+
+""" Calculating SIRS Radiation Estimates """
+
+import numpy as np
+from astral import sun
+from scipy.constants import Stefan_Boltzmann
+from datetime import datetime, timedelta
+import astral
+
+def numpy_to_datetime(date_time64_val):
```
Comment (AdamTheisen) [inline act/retrievals/sirs_calculations.py:10]: This function is already include in the datetime_utils.py moduledatetime64_to_datetime.  We should remove from here and use the existing function.

File: act/retrievals/sirs_calculations.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    '''
+
+    py_datetime_val = (
+        date_time64_val -
+        np.datetime64('1970-01-01T00:00:00')) / np.timedelta64(1, 's')
+
+    return py_datetime_val
+
+
+def calculate_sirs_variables(sirs_obj, met_obj, sirs_time='time', met_time='time', lat='lat', lon='lon',
```
Comment (AdamTheisen) [inline act/retrievals/sirs_calculations.py:36]: Would it make sense to split this out into different functions for each calculated variable?

Comment (AdamTheisen) [reply act/retrievals/sirs_calculations.py:36]: We also probably want to rename this to a more general name like radiation.py or something like that.  Might help with non-ARM lingo knowing users :)

File: act/retrievals/sirs_calculations.py (hunk 1 above)
Comment (ajsockol) [reply act/retrievals/sirs_calculations.py:10]: Okay I will change this!

File: act/retrievals/sirs_calculations.py (hunk 2 above)
Comment (ajsockol) [reply act/retrievals/sirs_calculations.py:36]: I don't know - some of the calculations are tied together (for example, you need the derived down short hemispheric value to get the irradiance difference and irradiance ratio), and they are all pretty closely related, so my first thought would be that it makes more sense to keep them all together. What do you think?

And, sounds good - I will change the name to something more generic.

**Comment by AdamTheisen** [comment]
Failed on some pep8 issues

./act/retrievals/sirs_calculations.py:10:1: E302 expected 2 blank lines, found 1
./act/retrievals/sirs_calculations.py:36:105: W291 trailing whitespace
./act/retrievals/sirs_calculations.py:37:5: E128 continuation line under-indented for visual indent
./act/retrievals/sirs_calculations.py:38:5: E128 continuation line under-indented for visual indent
./act/retrievals/sirs_calculations.py:39:5: E128 continuation line under-indented for visual indent
./act/retrievals/sirs_calculations.py:40:5: E128 continuation line under-indented for visual indent
./act/retrievals/sirs_calculations.py:40:5: E125 continuation line with same indent as next logical line
./act/retrievals/sirs_calculations.py:95:37: E231 missing whitespace after ','
./act/retrievals/sirs_calculations.py:101:67: E226 missing whitespace around arithmetic operator
./act/retrievals/sirs_calculations.py:101:116: E501 line too long (128 > 115 characters)
./act/retrievals/sirs_calculations.py:131:47: E261 at least two spaces before inline comment
./act/retrievals/sirs_calculations.py:132:58: E261 at least two spaces before inlin

**Comment by ajsockol** [comment]
Changed the name of the script to something more generic, and this final upload should (hopefully) have fixed all of the PEP8 issues. Still need to work out the kinks of using datetime64_to_datetime rather than numpy_to_datetime, but will add the changes here once I get it figured out!

**Comment by AdamTheisen** [comment]
@ajsockol , it looks like this changes the original datetime64 converter so the unit tests fail.  I'll take a look in the next couple days

**Comment by ajsockol** [comment]
Hey @AdamTheisen , Ken and I got some kinks ironed out this afternoon - these changes should work!

**Comment by AdamTheisen** [comment]
@ajsockol @kenkehoe would you be able to add  unit test for this?  The coverage would go down by 1% otherwise.  Thanks!

**Comment by ajsockol** [comment]
@AdamTheisen , unit test added!

**Comment by AdamTheisen** [comment]
@ajsockol Did you want to take one last look at this PR to make sure everything looks as expected?  Thanks!

**Pull request merged**